bitnami/wordpress/templates/deployment.yaml

```diff
@@ -45,6 +45,9 @@ spec:
         runAsUser: {{ .Values.securityContext.runAsUser }}
         fsGroup: {{ .Values.securityContext.fsGroup }}
       {{- end }}
+      {{- if .Values.initContainers }}
+        {{- include "wordpress.tplValue" (dict "value" .Values.initContainers "context" $) | nindent 6 }}
```

Comment:
This should be indented 8 spaces (instead of 6):
```diff
-          {{- include "wordpress.tplValue" (dict "value" .Values.initContainers "context" $) | nindent 6 }}
+       {{- include "wordpress.tplValue" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
```